updating fontawesome assets from v5.15.3 to v6.5.2

jkellz-dev opened the pull request:
To get some of the newer icons for the brands, I pulled the updated free FA icons. I'm not certain all of the files are necessary, but I put everything there just in case.

jkellz-dev committed:
updating fontawesome assets to v6

fncnt commented:
Thank you! Looks mostly good to me after checking the `sha512` sums of the files against the original archive from fontawesome.
It appears that you took `sprites/*.svg` and renamed them to match the `webfonts/*.svg` files I used from fontawesome `5.15.3`. 
In `6.5.2`, `webfonts/` doesn't contain any `.svg` files anymore, so I'd prefer if you could remove them in this PR too.

I'm happy to merge your PR afterwards.

fncnt commented on static/webfonts/fa-brands-400.svg:
This file doesn't exist anymore in fontawesome `6.5.2`. Please delete instead.

jkellz-dev replied:
removed

fncnt commented on static/webfonts/fa-regular-400.svg:
This file doesn't exist anymore in fontawesome `6.5.2`. Please delete instead.

fncnt commented on static/webfonts/fa-solid-900.svg:
This file doesn't exist anymore in fontawesome `6.5.2`. Please delete instead.

jkellz-dev commented:
Oh, yeah, I totally made an assumption without actually digging into it at all. My bad. I'll update right away.

jkellz-dev committed:
removing svg files

Sprite files are no longer used in the webfonts implementation.
Removing these incorrectly added files.

fncnt commented:
Thanks, merged!
